### DISCUSSION TITLE
php调用api使用流式上传文件后，为什么文件开头会有--------------------------Content-Disposition这样的字符串？

### DISCUSSION THREAD

**Discussion by caocaofff** [AlistGo/alist#7604] [Q&A]
调用接口已经返回200了，然后去alist刷新查看文件时，发现文件开头被填充了这些字符，导致文件损坏
![image](https://github.com/user-attachments/assets/ff5ec5a0-4df7-47db-bf73-811359838413)

**Comment by xhofe** [comment]
你的调用代码写的有问题。
